src/N3Store.js

```diff
@@ -316,7 +342,15 @@ export default class N3Store {
   // ### `removeMatches` removes all matching quads from the store
   // Setting any field to `undefined` or `null` indicates a wildcard.
   removeMatches(subject, predicate, object, graph) {
-    return this.remove(this.match(subject, predicate, object, graph));
+    const stream = new Readable({ objectMode: true });
```

Comment:
Since the old implementation of this.match was only being used here, I just inlined the old implementation.